src/vic3_world/buildings/buildings.cpp

```diff
@@ -5,6 +5,27 @@
 
 
 
+namespace
+{
+
+bool IsIndustrialBuilding(const vic3::Building& building)
+{
+   const std::string& type = building.GetType();
+
+   return type == "building_food_industry" || type == "building_textile_mills" ||
```

Comment:
TODO: I'm sure this can be automated somehow. Either from vic3 install or a config file. This looks like the type of thing one forgets to update after PDX introduces new buildings in some patch.

Reply:
#370